services/frontend/src/components/login/Login.js

```diff
@@ -101,11 +101,16 @@ const Login = ({ setToken }) => {
           }),
       }
       ).then(response => {
-          return response.json()
-      }).then(res => {
-          console.log(res)
-      })
-  }
+        return response.json()
+    }).then(res => {
+      if (res.status === 200) {
+        alert('Sign up successful')
+      } else {
+        let errorMessage = res.detail.msg || res.detail[0].msg
```

Comment:
In case the error message is a string there is not 'msg' key under 'detail' key. Fix this.